doc/newsfragments/1937_changed.dict_match_for_ignored_fields.rst

```diff
@@ -0,0 +1 @@
+In :py:meth:`dict.match <testplan.testing.multitest.result.DictNamespace.match>` assertion, the nested values under ignored keys are still compared one by one, so that 2 dictionaries can be displayed in a compact view on Web UI (comparable fields are placed in the same row).
```

Comment:
When :py:meth:`dict.match <testplan.testing.multitest.result.DictNamespace.match>` ``exclude_keys`` has nested dict, align the values under the same key between actual and expected on Web UI.